src/components/pages/Project.vue

```diff
@@ -0,0 +1,147 @@
+                {{project.type === 'donate' ? 'Spendenseite' : 'Betriebsseite'}}
+            </h1>
+            <h1 class="display-1 hidden-md-and-up font-weight-thin white--text ">
+                {{project.type === 'donate' ? 'Spendenseite' : 'Betriebsseite'}}
+            </h1>
+
+        </div>
+        <v-container>
+            <v-row>
+                <v-col>
+                    <v-card class="projectBox" elevation="4">
+                        <v-card-title class="font-weight-light display-1">
+                            {{project.name}}
+                        </v-card-title>
+                        <v-card-text>
+                            <div class="mb-4">
+                                <h3>Adresse</h3>
+                                {{project.street}}<br>
+                                {{project.zip}} {{project.city}}
+                            </div>
+                            <v-card elevation="7" class="py-6 text-center">
+                                <v-row>
+                                    <v-col>
+                                        <h4 class="headline">Gesammelt</h4>
+                                        <h1 class="display-2 font-weight-thin">{{project.donationGoal.reached.toFixed(2)}}€</h1>
+
+                                    </v-col>
+                                    <v-col>
+                                        <h4 class="headline">Ziel</h4>
+                                        <h1 class="display-2 font-weight-thin">{{project.donationGoal.total.toFixed(2)}}€</h1>
+
+                                    </v-col>
+                                </v-row>
+                                <h2>{{(project.donationGoal.reached/project.donationGoal.total*100).toFixed(1)}}%</h2>
+                                <v-progress-linear
+                                        color="secondary"
+                                        height="15"
+                                        :value="project.donationGoal.reached/project.donationGoal.total*100"
+                                        striped
+                                ></v-progress-linear>
+                                <br>
+                                <hr>
+                                <currency-input class="mt-3 headline" v-model="donationValue" /><br>
+                                <v-btn class="mt-2 btn-hover color-9" dark>Betrag Spenden</v-btn>
+                            </v-card>
+
+
+                        </v-card-text>
+                        <v-card-actions>
+                            <a :href="project.homepage">
+                                <v-btn outlined>Webseite besuchen</v-btn>
+                            </a>
+
+                        </v-card-actions>
+                    </v-card>
+                </v-col>
+                <v-col>
+                    <v-card>
+                        <v-img src="https://i.imgur.com/EJOjIMC.jpg" />
```

Comment:
ist das "Projektabhängig"? dann wäre das ja gut wenns im project-object drin wäre. 
wenn nicht, sry

Reply:
Noch nicht aber was ja nicht ist das kann ja noch werden. Demnach you're right!

Reply:
Sollte zu Ihrer Zufriedenheit gefixt worden sein tun.